.repo-metadata.json

```diff
@@ -1,19 +1,19 @@
 {
   "api_shortname": "pubsub",
   "name_pretty": "Cloud Pub/Sub",
-  "api_reference": "https://cloud.google.com/pubsub/",
   "product_documentation": "https://cloud.google.com/pubsub/docs/",
-  "client_documentation": "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest/history",
   "api_description": "is designed to provide reliable, many-to-many, asynchronous messaging between applications. Publisher applications can send messages to a topic and other applications can subscribe to that topic to receive the messages. By decoupling senders and receivers, Google Cloud Pub/Sub allows developers to communicate between independently written applications.",
-  "issue_tracker": "https://issuetracker.google.com/savedsearches/559741",
+  "client_documentation": "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest/history",
   "release_level": "stable",
+  "transport": "both",
   "language": "java",
   "repo": "googleapis/java-pubsub",
   "repo_short": "java-pubsub",
   "distribution_name": "com.google.cloud:google-cloud-pubsub",
-  "codeowner_team": "@googleapis/api-pubsub",
   "api_id": "pubsub.googleapis.com",
   "library_type": "GAPIC_COMBO",
   "requires_billing": true,
-  "recommended_package": "com.google.cloud.pubsub.v1"
+  "api_reference": "https://cloud.google.com/pubsub/",
```

Comment:
I realized this and the "client_documentation" links were from the previous version, but "https://cloud.google.com/java/docs/reference/google-cloud-pubsub/latest/com.google.cloud.pubsub.v1" is likely a better URL to use for "api_reference".